esmvalcore/config-developer.yml

```
@@ -194,3 +194,14 @@ CESM:
   output_file: '{project}_{dataset}_{case}_{gcomp}_{scomp}_{type}_{mip}_{short_name}'
   cmor_type: 'CMIP6'
   cmor_default_table_prefix: 'CMIP6_'
+
+ACCESS:
+  cmor_strict: false
+  input_dir:
+    default:
+      - '{institute}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
+  input_file:
+    default: '{sub_dataset}.{special_attr}-*.nc'
+  output_file: '{institute}_{sub_dataset}_{special_attr}_{short_name}'
```

```suggestion
  output_file: '{project}_{dataset}_{mip}_{exp}_{institute}_{sub_dataset}_{special_attr}_{short_name}'
```

This makes it more aligned with CMIP6 output and also ensures that no data is accidentally overwritten (which can lead to bugs that are really hard to debug). Please adapt in case you follow my comment above and get rid of `institute`.